### DISCUSSION TITLE
Moving pieces out of their container

### DISCUSSION THREAD

**Discussion by lucawittorf** [flbulgarelli/headbreaker#67] [Q&A] [answered]
Is it possible to store the pieces at the border of the container or in another one when opening/starting the puzzle? I would like to have one container where the pieces are stored and another container where i can do/play the puzzle.

**Comment by flbulgarelli** [comment]
Hi @lucawittorf! 

:wave: Hi! The short answer is that there is no explicit support for this feature, and it probably won't be since it is currently out of the scope of the framework. You might implement something similar to that on top of the shuffling capabilities, though. See https://github.com/flbulgarelli/headbreaker/issues/17.

**Comment by lucawittorf** [comment, marked as answer]
**@flbulgarelli** Hi, thanks for your answer. I used something similar here. 
There is another thing i would like to know. When moving a group of puzzles, only the piece you grab wouldnt move out of the canvas container: 

![image](https://github.com/flbulgarelli/headbreaker/assets/121250908/79524399-3847-481d-9dad-825ed33e61b7)

The other pieces of that group would disappear if you move the group more. Is this a bug and not being fixed right now or could i prevent this somehow?

**Reply by flbulgarelli** [reply]
Hi! The offstage prevention, as you mentioned, just works for the current piece and it is not intended to prevent that other pieces go off-stage out of the box. For more details, check https://github.com/flbulgarelli/headbreaker/issues/25 and https://github.com/flbulgarelli/headbreaker/issues/2.
